<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8">
  <title>Datasets - NBench</title>
  <meta name="author" content="Chichau Miau">

  
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://chichaumiau.github.io/nbench/datasets/">
  <link href="/favicon.png" type="image/png" rel="icon">
  <link href="/nbench/atom.xml" rel="alternate" title="NBench" type="application/atom+xml">

  <!-- http://opengraphprotocol.org/ -->
  <meta name="twitter:card" content="summary_large_image">
  <meta property="og:type" content="website">
  <meta property="og:url" content="http://chichaumiau.github.io/nbench/datasets/">
  <meta property="og:title" content="Datasets - NBench">
  

  <script src="/nbench/assets/bootstrap/dist/js/jquery-3.2.1.min.js"></script>
<script src="/nbench/assets/bootstrap/dist/js/bootstrap.min.js"></script>

<link href="/nbench/assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css">
<link href="/nbench/assets/bootstrap/dist/css/stylish-portfolio.css" rel="stylesheet" type = "text/css">
<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
<link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">


  
  <link href="/nbench/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">

  

</head>

  <body   >
    <a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>
    <div id="wrap">
      <header role="banner">
        <nav class="navbar navbar-custom" role="navigation">
	
    <div class="container">
        <div class="navbar-header">
<!--
            <button type="button" class="navbar-toggle" title="toggle navbar" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
-->
            <img src="/nbench/images/logo.jpg" style='max-height:50px;' alt="logo">
            <a class="navbar-brand" href="/nbench/">NBench</a>
        </div>

        <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav">
                <li >
                    <a href="/nbench/news">News</a>
                </li>
                <li >
                    <a href="/nbench/results">Results</a>
                </li>
                <li class="active">
                    <a href="/nbench/datasets">Datasets</a>
                </li>
                <li >
                    <a href="/nbench/programs">Prediction Programs</a>
                </li>
                <li >
                    <a href="/nbench/discussion">Discussion</a>
                </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li>
                    <a class="subscribe-rss" href="/nbench/atom.xml" title="subscribe via RSS">
                        <span class="visible-xs">RSS</span>
                        <img class="hidden-xs" src="/nbench/images/rss.png" alt="RSS">
                    </a>
                </li>
                
            </ul>
            
                <form class="navbar-form navbar-right" action="https://www.google.com/search" method="GET">
                    <input type="hidden" name="sitesearch" value="chichaumiau.github.io/nbench">
                    <div class="form-group">
                        <input class="form-control" type="text" name="q" placeholder="Search">
                    </div>
                </form>
            
        </div>
    </div>
</nav>


<a id="menu-toggle" href="#" class="btn btn-dark btn-lg toggle"><i class="fa fa-bars"></i></a>
<nav id="sidebar-wrapper">
	<ul class="sidebar-nav">
		<a id="menu-close" href="#" class="btn btn-dark btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
		<li class="sidebar-brand"><a href="https://chichaumiau.github.io/nbench" onclick=$("#menu-close").click();>NBench</a></li>
		<li><a href="#top" onclick=$("#menu-close").click();>Home</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/news" onclick=$("#menu-close").click();>News</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/results" onclick=$("#menu-close").click();>Results</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/datasets" onclick=$("#menu-close").click();>Datasets</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/programs" onclick=$("#menu-close").click();>Programs</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/discussion" onclick=$("#menu-close").click();>Discussion</a></li>
		<li><a href="https://github.com/RNA-Puzzles" onclick=$("#menu-close").click();>Github</a></li>
	</ul>
</nav>


      </header>
      <div id="main" role="main" class="container">
        <div id="content">
          <div class="row">
  <div class="page-content col-md-9">
    <article role="article">
      
        <header class="page-header">
          
            <p class="meta text-muted text-uppercase">












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2018-01-30T14:38:00+00:00"  data-updated="true" itemprop="datePublished dateCreated">ordinal</time></p>
          
          
            <h1 class="entry-title">
              Datasets
              
            </h1>
          
        </header>
      
      <div class="entry-content">
        <div align="center">
<table border="1">
<tr>
<td>Year</td>
<td>Program</td>
<td>Dataset</td>
<td>Protein#</td>
<td>cutoff(&Aring;)<sup>(a)</sup></td>
<td>resolution(&Aring;)</td>
<td>SeqId(%)<sup>(b)</sup></td>
<td>StrId<sup>(c)</sup></td>
<td>Update#<sup>(d)</sup></td>
</tr>
<tr><td colspan="9"><h3>RNA binding proteins</h3></td></tr>
<tr>
<td>2006</td>
<td>BindN<sup>1</sup></td>
<td>BindN_R107</td>
<td><b><a href="/nbench/dataset/BindN_R107.old">107</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/BindN_R107.txt">95</a></b></td>
</tr><tr>
<td>2008</td>
<td>DR_bind1<sup>2</sup></td>
<td>DR_bind1_R69</td>
<td><b><a href="/nbench/dataset/DR_bind1_R69.old">69</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>CATH</td>
<td><b><a href="/nbench/dataset/DR_bind1_R69.txt">69</a></b></td>
</tr><tr>
<td>2014</td>
<td>DR_bind1<sup>3</sup></td>
<td>DR_bind1_R81</td>
<td><b><a href="/nbench/dataset/DR_bind1_R81.old">81</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>CATH</td>
<td><b><a href="/nbench/dataset/DR_bind1_R81.txt">79</a></b></td>
</tr><tr>
<td>2006</td>
<td>KYG<sup>4</sup></td>
<td>KYG_R86</td>
<td><b><a href="/nbench/dataset/KYG_R86.old">86</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>50</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/KYG_R86.txt">85</a></b></td>
</tr><tr>
<td>2007</td>
<td>PPRInt<sup>5</sup></td>
<td>PPRInt_R86</td>
<td><b><a href="/nbench/dataset/PPRInt_R86.old">86</a></b></td>
<td>3.5</td>
<td>3</td>
<td>70</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/PPRInt_R86.txt">83</a></b></td>
</tr><tr>
<td>2010</td>
<td>PRNA<sup>6</sup></td>
<td>PRNA_R205</td>
<td><b><a href="/nbench/dataset/PRNA_R205.old">205</a></b></td>
<td>3.5</td>
<td>3</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/PRNA_R205.txt">189</a></b></td>
</tr><tr>
<td>2011</td>
<td>SRCPred<sup>7</sup></td>
<td>SRCPred_R160</td>
<td><b><a href="/nbench/dataset/SRCPred_R160.old">160</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/SRCPred_R160.txt">124</a></b></td>
</tr><tr>
<td>2011</td>
<td>PRBR<sup>8</sup></td>
<td>PRBR_R180</td>
<td><b><a href="/nbench/dataset/PRBR_R180.old">180</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/PRBR_R180.txt">142</a></b></td>
</tr><tr>
<td>2006</td>
<td>RNABindR<sup>9</sup></td>
<td>RNABindR_R106</td>
<td><b><a href="/nbench/dataset/RNABindR_R106.old">106</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R106.txt">100</a></b></td>
</tr><tr>
<td>2007</td>
<td>RNABindR<sup>10</sup></td>
<td>RNABindR_R109</td>
<td><b><a href="/nbench/dataset/RNABindR_R109.old">109</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R109.txt">100</a></b></td>
</tr><tr>
<td>2007</td>
<td>RNABindR<sup>10</sup></td>
<td>RNABindR_R144</td>
<td><b><a href="/nbench/dataset/RNABindR_R144.old">144</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R144.txt">137</a></b></td>
</tr><tr>
<td>2007</td>
<td>RNABindR<sup>10</sup></td>
<td>RNABindR_R147</td>
<td><b><a href="/nbench/dataset/RNABindR_R147.old">147</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R147.txt">138</a></b></td>
</tr><tr>
<td>2007</td>
<td>RNABindR<sup>10</sup></td>
<td>RNABindR_R198</td>
<td><b><a href="/nbench/dataset/RNABindR_R198.old">198</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R198.txt">187</a></b></td>
</tr><tr>
<td>2014</td>
<td>RNABindRPlus<sup>11</sup></td>
<td>RNABindR_R44</td>
<td><b><a href="/nbench/dataset/RNABindR_R44.txt">44</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>40</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R44.txt">44</a></b></td>
</tr><tr>
<td>2014</td>
<td>RNABindRPlus<sup>11</sup></td>
<td>RNABindR_R111</td>
<td><b><a href="/nbench/dataset/RNABindR_R111.old">111</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RNABindR_R111.txt">101</a></b></td>
</tr><tr>
<td>2012</td>
<td>meta2<sup>12</sup></td>
<td>meta2_R44</td>
<td><b><a href="/nbench/dataset/meta2_R44.txt">44</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>40</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/meta2_R44.txt">44</a></b></td>
</tr><tr>
<td>2014</td>
<td>aaRNA<sup>13</sup></td>
<td>aaRNA_R67</td>
<td><b><a href="/nbench/dataset/aaRNA_R67.txt">67</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/aaRNA_R67.txt">67</a></b></td>
</tr><tr>
<td>2014</td>
<td>aaRNA<sup>13</sup></td>
<td>aaRNA_R141</td>
<td><b><a href="/nbench/dataset/aaRNA_R141.old">141</a></b></td>
<td>3.5</td>
<td>3</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/aaRNA_R141.txt">136</a></b></td>
</tr><tr>
<td>2014</td>
<td>aaRNA<sup>13</sup></td>
<td>aaRNA_R205</td>
<td><b><a href="/nbench/dataset/aaRNA_R205.old">205</a></b></td>
<td>3.5</td>
<td>3</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/aaRNA_R205.txt">200</a></b></td>
</tr><tr>
<td>2015</td>
<td>RBscore</td>
<td>RBscore_R130</td>
<td><b><a href="/nbench/dataset/RBscore_R130.txt">130</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>TMscore&lt;0.7</td>
<td><b><a href="/nbench/dataset/RBscore_R130.txt">130</a></b></td>
</tr><tr>
<td>2015</td>
<td>RBscore</td>
<td>RBscore_R116</td>
<td><b><a href="/nbench/dataset/RBscore_R116.txt">116</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>TMscore&lt;0.7</td>
<td><b><a href="/nbench/dataset/RBscore_R116.txt">116</a></b></td>
</tr><tr>
<td>2011</td>
<td><sup>14</sup></td>
<td>Sungwook_R267</td>
<td><b><a href="/nbench/dataset/Sungwook_R267.old">267</a></b></td>
<td>3.5</td>
<td>3</td>
<td>60</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Sungwook_R267.txt">178</a></b></td>
</tr><tr>
<td>2011</td>
<td><sup>14</sup></td>
<td>Sungwook_R727</td>
<td><b><a href="/nbench/dataset/Sungwook_R727.old">727</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Sungwook_R727.txt">574</a></b></td>
</tr><tr>
<td>2011</td>
<td><sup>14</sup></td>
<td>Sungwook_R3149</td>
<td><b><a href="/nbench/dataset/Sungwook_R3149.old">3149</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Sungwook_R3149.txt">2632</a></b></td>
</tr><tr>
<td>2015</td>
<td>After2014</td>
<td>New_R15</td>
<td><b><a href="/nbench/dataset/New_R15.txt">15</a></b></td>
<td></td>
<td></td>
<td>25</td>
<td>TMscore<0.7</td>
<td><b><a href="/nbench/dataset/New_R15.txt">15</a></b></td>
</tr>
<tr><td colspan="9"><h3>DNA binding proteins</h3></td></tr>
<tr>
<td>2006</td>
<td>BindN<sup>1</sup></td>
<td>BindN_D62</td>
<td><b><a href="/nbench/dataset/BindN_D62.old">62</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/BindN_D62.txt">66</a></b></td>
</tr><tr>
<td>2010</td>
<td><sup>15</sup></td>
<td>ProteDNA_D253</td>
<td><b><a href="/nbench/dataset/ProteDNA_D253.txt">253</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>20</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/ProteDNA_D253.txt">253</a></b></td>
</tr><tr>
<td>2008</td>
<td>Pro-dna<sup>16</sup></td>
<td>Pro-dna_D99</td>
<td><b><a href="/nbench/dataset/Pro-dna_D99.old">99PDB</a></b></td>
<td>3.5</td>
<td>3</td>
<td>20</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Pro-dna_D99.txt">188</a></b></td>
</tr><tr>
<td>1999</td>
<td><sup>17</sup></td>
<td>Hidetoshi_D52</td>
<td><b><a href="/nbench/dataset/Hidetoshi_D52.old">52</a></b></td>
<td>3.5</td>
<td>3.2</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Hidetoshi_D52.txt">49</a></b></td>
</tr><tr>
<td>2008</td>
<td><sup>18</sup></td>
<td>Shandar_D140</td>
<td><b><a href="/nbench/dataset/Shandar_D140.old">140</a></b></td>
<td>3.5</td>
<td>2.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Shandar_D140.txt">138</a></b></td>
</tr><tr>
<td>2003</td>
<td><sup>19</sup></td>
<td>Susan_D56</td>
<td><b><a href="/nbench/dataset/Susan_D56.old">56</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>CATH</td>
<td><b><a href="/nbench/dataset/Susan_D56.txt">54</a></b></td>
</tr><tr>
<td>2008</td>
<td>DBD-Hunter<sup>20</sup></td>
<td>DBD-Hunter_D179</td>
<td><b><a href="/nbench/dataset/DBD-Hunter_D179.old">179</a></b></td>
<td>3.5</td>
<td>3</td>
<td>35</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/DBD-Hunter_D179.txt">177</a></b></td>
</tr><tr>
<td>2001</td>
<td><sup>21</sup></td>
<td>Luscombe_D129</td>
<td><b><a href="/nbench/dataset/Luscombe_D129.old">129</a></b></td>
<td>3.5</td>
<td>3</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/Luscombe_D129.txt">182</a></b></td>
</tr><tr>
<td>2000</td>
<td>DBindR<sup>22</sup></td>
<td>DBindR_D374</td>
<td><b><a href="/nbench/dataset/DBindR_D374.old">374</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/DBindR_D374.txt">329</a></b></td>
</tr><tr>
<td>2007</td>
<td>DISPLAR<sup>23</sup></td>
<td>DISPLAR_D428</td>
<td><b><a href="/nbench/dataset/DISPLAR_D428.old">428</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>50</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/DISPLAR_D428.txt">390</a></b></td>
</tr><tr>
<td>2010</td>
<td>DNABINDPROT<sup>24</sup></td>
<td>DNABINDPROT_D54</td>
<td><b><a href="/nbench/dataset/DNABINDPROT_D54.old">54</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/DNABINDPROT_D54.txt">50</a></b></td>
</tr><tr>
<td>2013</td>
<td>PreDNA<sup>25</sup></td>
<td>PreDNA_D224</td>
<td><b><a href="/nbench/dataset/PreDNA_D224.old">224</a></b></td>
<td>3.5</td>
<td>3</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/PreDNA_D224.txt">216</a></b></td>
</tr><tr>
<td>2015</td>
<td>RBscore</td>
<td>RBscore_D381</td>
<td><b><a href="/nbench/dataset/RBscore_D381.txt">381</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RBscore_D381.txt">381</a></b></td>
</tr><tr>
<td>2011</td>
<td>metaDBSite<sup>26</sup></td>
<td>metaDBSite_D232</td>
<td><b><a href="/nbench/dataset/metaDBSite_D232.old">232</a></b></td>
<td>3.5</td>
<td>3</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/metaDBSite_D232.txt">225</a></b></td>
</tr><tr>
<td>2011</td>
<td>metaDBSite<sup>26</sup></td>
<td>metaDBSite_D316</td>
<td><b><a href="/nbench/dataset/metaDBSite_D316.old">316</a></b></td>
<td>3.5</td>
<td>3</td>
<td>30</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/metaDBSite_D316.txt">308</a></b></td>
</tr><tr>
<td>2009</td>
<td>SDCPred<sup>27</sup></td>
<td>SDCPred_D159</td>
<td><b><a href="/nbench/dataset/SDCPred_D159.old">159</a></b></td>
<td>3.5</td>
<td>2.5</td>
<td>25</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/SDCPred_D159.txt">158</a></b></td>
</tr><tr>
<td>2015</td>
<td>After2014</td>
<td>New_D31</td>
<td><b><a href="/nbench/dataset/New_D31.txt">31</a></b></td>
<td></td>
<td></td>
<td>25</td>
<td>TMscore<0.7</td>
<td><b><a href="/nbench/dataset/New_D31.txt">31</a></b></td>
</tr>

<tr><td colspan="9"><h3>Sum</h3></td></tr>
<tr>
<td>2015</td>
<td>RBscore</td>
<td>RBscore_P627</td>
<td><b><a href="/nbench/dataset/RBscore_P627.txt">627</a></b></td>
<td>3.5</td>
<td>3.5</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/RBscore_P627.txt">627</a></b></td>
</tr><tr>
<td>2000</td>
<td>ALL</td>
<td>All_P5114</td>
<td><b><a href="/nbench/dataset/All_P5114.txt">5114</a></b></td>
<td>3.5</td>
<td>NA</td>
<td>NA</td>
<td>NA</td>
<td><b><a href="/nbench/dataset/All_P5114.txt">5114</a></b></td>
</tr>
<tr>
<td colspan="9"><b><a href="/nbench/dataset/badcases">Excluded cases</a></b></td>
</tr>
</table>
</div>


<p>(a)distance cutoff used to define binding sites<br>
(b)sequence identity<br>
(c)structural identity<br>
(d)unreasonable cases removed<br></p>

<h2>References:</h2>

<ol>
<li>Wang L, Brown SJ. BindN: a web-based tool for efficient prediction of DNA and RNA binding sites in amino acid sequences. Nucleic Acids Res. 2006;34(Web Server issue):W243-8. doi:10.1093/nar/gkl298.</li>
<li>Chen YC, Lim C. Predicting RNA-binding sites from the protein structure based on electrostatics, evolution and geometry. Nucleic Acids Res. 2008;36(5):e29. doi:10.1093/nar/gkn008.</li>
<li>Chen YC, Sargsyan K, Wright JD, Huang Y-S, Lim C. Identifying RNA-binding residues based on evolutionary conserved structural and energetic features. Nucleic Acids Res. 2014;42(3):e15. doi:10.1093/nar/gkt1299.</li>
<li>Kim OTP, Yura K, Go N. Amino acid residue doublet propensity in the protein-RNA interface and its application to RNA interface prediction. Nucleic Acids Res. 2006;34(22):6450-60. doi:10.1093/nar/gkl819.</li>
<li>Kumar M, Gromiha MM, Raghava GPS. Prediction of RNA binding sites in a protein using SVM and PSSM profile. 2007:189-194. doi:10.1002/prot.21677.</li>
<li>Liu Z-P, Wu L-Y, Wang Y, Zhang X-S, Chen L. Prediction of protein-RNA binding sites by a random forest method with combined features. Bioinformatics 2010;26(13):1616-22. doi:10.1093/bioinformatics/btq253.</li>
<li>Fernandez M, Kumagai Y, Standley DM, Sarai A, Mizuguchi K, Ahmad S. Prediction of dinucleotide-specific RNA-binding sites in proteins. BMC Bioinformatics 2011;12 Suppl 1(Suppl 13):S5. doi:10.1186/1471-2105-12-S13-S5.</li>
<li>Ma X, Guo J, Wu J, et al. Prediction of RNA-binding residues in proteins from primary sequence using an enriched random forest model with a novel hybrid feature. Proteins 2011;79(4):1230-9. doi:10.1002/prot.22958.</li>
<li>Terribilini M, Lee J, Yan C, Jernigan RL, Honavar V, Dobbs D. Prediction of RNA binding sites in proteins from amino acid sequence. 2006:1450-1462. doi:10.1261/rna.2197306.and.</li>
<li>Terribilini M, Sander JD, Lee J-H, et al. RNABindR: a server for analyzing and predicting RNA-binding sites in proteins. Nucleic Acids Res. 2007;35(Web Server issue):W578-84. doi:10.1093/nar/gkm294.</li>
<li>Walia RR, Xue LC, Wilkins K, El-Manzalawy Y, Dobbs D, Honavar V. RNABindRPlus: A Predictor that Combines Machine Learning and Sequence Homology-Based Methods to Improve the Reliability of Predicted RNA-Binding Residues in Proteins. PLoS One 2014;9(5):e97725. doi:10.1371/journal.pone.0097725.</li>
<li>Puton T, Kozlowski L, Tuszynska I, Rother K, Bujnicki JM. Computational methods for prediction of protein-RNA interactions. J. Struct. Biol. 2012;179(3):261-8. doi:10.1016/j.jsb.2011.10.001.</li>
<li>Li S, Yamashita K, Amada KM, Standley DM. Quantifying sequence and structural features of protein-RNA interactions. Nucleic Acids Res. 2014;42(15):10086-98. doi:10.1093/nar/gku681.</li>
<li>Choi S, Han K. Prediction of RNA-binding amino acids from protein and RNA sequences. BMC Bioinformatics 2011;12 Suppl 1(Suppl 13):S7. doi:10.1186/1471-2105-12-S13-S7.</li>
<li>Huang Y-F, Chiu L-Y, Huang C-KC-C. Predicting RNA-binding residues from evolutionary information and sequence conservation. BMC Genomics 2010;11 Suppl 4(Suppl 4):S2. doi:10.1186/1471-2164-11-S4-S2.</li>
<li>Nitin Bhardwaj HL. Residue-Level Prediction of DNA-Binding Sites and its Application on DNA-Binding Protein Predictions. FEBS Lett 2008;581(5):1058-1066.</li>
<li>Kono H, Sarai A. Structure-Based Prediction of DNA Target Sites. 1999;131(November 1998):114-131.</li>
<li>Ahmad S, Keskin O, Sarai A, Nussinov R. Protein-DNA interactions: structural, thermodynamic and clustering patterns of conserved residues in DNA-binding proteins. Nucleic Acids Res. 2008;36(18):5922-32. doi:10.1093/nar/gkn573.</li>
<li>Jones S. Using electrostatic potentials to predict DNA-binding sites on DNA-binding proteins. Nucleic Acids Res. 2003;31(24):7189-7198. doi:10.1093/nar/gkg922.</li>
<li>Gao M, Skolnick J. DBD-Hunter: a knowledge-based method for the prediction of DNA-protein interactions. Nucleic Acids Res. 2008;36(12):3978-92. doi:10.1093/nar/gkn332.</li>
<li>Luscombe NM, Laskowski R a, Thornton JM. Amino acid-base interactions: a three-dimensional analysis of protein-DNA interactions at an atomic level. Nucleic Acids Res. 2001;29(13):2860-74. Available at: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=55782&amp;amp;tool=pmcentrez&amp;amp;rendertype=abstract.">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=55782&amp;amp;tool=pmcentrez&amp;amp;rendertype=abstract.</a></li>
<li>Wu J, Liu H, Duan X, et al. Prediction of DNA-binding residues in proteins from amino acid sequences using a random forest model with a hybrid feature. Bioinformatics 2009;25(1):30-5. doi:10.1093/bioinformatics/btn583.</li>
<li>Tjong H, Zhou H-X. DISPLAR: an accurate method for predicting DNA-binding sites on protein surfaces. Nucleic Acids Res. 2007;35(5):1465-77. doi:10.1093/nar/gkm008.</li>
<li>Ozbek P, Soner S, Erman B, Haliloglu T. DNABINDPROT: fluctuation-based predictor of DNA-binding residues within a network of interacting residues. Nucleic Acids Res. 2010;38(Web Server issue):W417-23. doi:10.1093/nar/gkq396.</li>
<li>Li T, Li Q-Z, Liu S, Fan G-L, Zuo Y-C, Peng Y. PreDNA: accurate prediction of DNA-binding sites in proteins by integrating sequence and geometric structure information. Bioinformatics 2013;29(6):678-85. doi:10.1093/bioinformatics/btt029.</li>
<li>Si J, Zhang Z, Lin B, Schroeder M, Huang B. MetaDBSite: a meta approach to improve protein DNA-binding sites prediction. BMC Syst. Biol. 2011;5 Suppl 1(Suppl 1):S7. doi:10.1186/1752-0509-5-S1-S7.</li>
<li>Andrabi M, Mizuguchi K, Sarai A, Ahmad S. Prediction of mono- and di-nucleotide-specific DNA-binding sites in proteins using neural networks. BMC Struct. Biol. 2009;9:30. doi:10.1186/1472-6807-9-30.</li>
</ol>


      </div>
      
        <footer>
          <p class="meta text-muted">
            
            












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2018-01-30T14:38:00+00:00"  data-updated="true" itemprop="datePublished dateCreated">ordinal</time>
            
          </p>
          
            <div class="sharing">
  
  <a href="//twitter.com/share" class="twitter-share-button" data-url="http://chichaumiau.github.io/nbench/datasets/index.html" data-via="chichaumiau" data-counturl="http://chichaumiau.github.io/nbench/datasets/index.html" >Tweet</a>
  
  
  
</div>

          
        </footer>
      
    </article>
    
      <section>
        <h2>Comments</h2>
        <div id="disqus_thread" aria-live="polite"><noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</div>
      </section>
    
  </div>

  
    <aside class="sidebar col-md-3">
      
        <section>
  <h1>NBench</h1>
  <p>Benchmark of nucleic acid binding site prediction methods.</p>
	<a href="https://github.com/chichaumiau"><i class="fa fa-github-square fa-fw fa-2x"></i></a>
	<a href="mailto:chichaumiau@gmail.com"><i class="fa fa-envelope-square fa-fw fa-2x"></i></a>
	<a href="https://twitter.com/chichaumiau"><i class="fa fa-twitter-square fa-fw fa-2x"></i></a>     
	<a href="https://plus.google.com/u/0/104588354121735738405"><i class="fa fa-google-plus-square fa-fw fa-2x"></i></a>
	<a href=""><i class="fa fa-coffee fa-fw fa-2x"></i></a>
</section>
<section class="panel panel-info">
  <div class="panel-heading">
    <h3 class="panel-title">Recent Posts</h3>
  </div>
  
  <div id="recent_posts" class="list-group">
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/moonlighting-proteins/">Moonlighting Proteins</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/nucleic-acid-binding-energy-funnel-on-protein-surface/">Nucleic Acid Binding Energy Funnel on Protein Surface</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/risky-non-orthogonal-features/">Risky Non-orthogonal Features</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/theoretical-defect-of-slide-window/">Theoretical Defect of Slide-window</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/discriminate-rna-binding-residues-from-dna-binding-residues/">Discriminate RNA Binding Residues From DNA Binding Residues</a>
    
  </div>
</section>

<section class="panel panel-info clearfix">
  <div class="panel-heading">
      <h3 class="panel-title">GitHub Repos</h3>
  </div>
  <div class="list-group" id="gh_repos">
    <p class="loading">Status updating...</p>
  </div>
  
    <div class="gh-profile-link pull-right text-muted">
      <a href="https://github.com/chichaumiau">@chichaumiau</a> on GitHub
    </div>
  
  <script type="text/javascript">
    $(document).ready(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'chichaumiau',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/nbench/javascripts/github.js" type="text/javascript"> </script>
</section>

<section class="panel panel-info clearfix">
  <div class="panel-heading">
      <h3 class="panel-title">Categories</h3>
  </div>
    <div class='list-group-item'><a href='/nbench/blog/categories/discussion/'>discussion (6)</a></div>
<div class='list-group-item'><a href='/nbench/blog/categories/table/'>table (1)</a></div>


</section>

<section class="googleplus panel panel-default">
  <div class="panel-body">
    <h2>
      <a href="https://plus.google.com/104588354121735738405?rel=author">
        <img src="http://www.google.com/images/icons/ui/gprofile_button-32.png" width="32" height="32">
        Google+
      </a>
    </h2>
  </div>
</section>





      
    </aside>
  
</div>

        </div>
      </div>
    </div>
    <footer role="contentinfo"><div class="container">
    <p class="text-muted credits" align="center">
  Copyright &copy; 2018 - <a href="mailto:chichaumiau@gmail.com">Chichau Miau</a>
  <a id="to-top" href="#top" class="btn btn-dark btn-lg"><i class="fa fa-chevron-up fa-fw fa-1x"></i></a>
</p>

</div>
</footer>
    

<script type="text/javascript">
      var disqus_shortname = 'rna-puzzles';
      
        
        // var disqus_developer = 1;
        var disqus_identifier = 'http://chichaumiau.github.io/nbench/datasets/index.html';
        var disqus_url = 'http://chichaumiau.github.io/nbench/datasets/index.html';
        var disqus_script = 'embed.js';
      
    (function () {
      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
      dsq.src = '//' + disqus_shortname + '.disqus.com/' + disqus_script;
      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    }());
</script>







  <script type="text/javascript">
    (function(){
      var twitterWidgets = document.createElement('script');
      twitterWidgets.type = 'text/javascript';
      twitterWidgets.async = true;
      twitterWidgets.src = '//platform.twitter.com/widgets.js';
      document.getElementsByTagName('head')[0].appendChild(twitterWidgets);
    })();
  </script>


<script src="/nbench/assets/bootstrap/dist/js/bootstrap.min.js"></script>
<script src="/nbench/javascripts/modernizr.js"></script>


<!-- Custom Theme JavaScript -->
<script>
// Closes the sidebar menu
$("#menu-close").click(function(e) {
	e.preventDefault();
	$("#sidebar-wrapper").toggleClass("active");
});
// Opens the sidebar menu
$("#menu-toggle").click(function(e) {
	e.preventDefault();
	$("#sidebar-wrapper").toggleClass("active");
});
// Scrolls to the selected menu item on the page
$(function() {
	$('a[href*=#]:not([href=#],[data-toggle],[data-target],[data-slide])').click(function() {
		if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
			var target = $(this.hash);
			target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
			if (target.length) {
				$('html,body').animate({
					scrollTop: target.offset().top
				}, 1000);
				return false;
			}
		}
	});
});
//#to-top button appears after scrolling
var fixed = false;
$(document).scroll(function() {
	if ($(this).scrollTop() > 250) {
		if (!fixed) {
			fixed = true;
			// $('#to-top').css({position:'fixed', display:'block'});
			$('#to-top').show("slow", function() {
				$('#to-top').css({
					position: 'fixed',
					display: 'block'
				});
			});
		}
	} else {
		if (fixed) {
			fixed = false;
			$('#to-top').hide("slow", function() {
				$('#to-top').css({
					display: 'none'
				});
			});
		}
	}
});
</script>


  </body>
</html>
