<!DOCTYPE html>
<html class="no-js" lang="en">
<head>
  <meta charset="utf-8">
  <title>Programs - NBench</title>
  <meta name="author" content="Chichau Miau">

  
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://chichaumiau.github.io/nbench/programs/">
  <link href="/favicon.png" type="image/png" rel="icon">
  <link href="/nbench/atom.xml" rel="alternate" title="NBench" type="application/atom+xml">

  <!-- http://opengraphprotocol.org/ -->
  <meta name="twitter:card" content="summary_large_image">
  <meta property="og:type" content="website">
  <meta property="og:url" content="http://chichaumiau.github.io/nbench/programs/">
  <meta property="og:title" content="Programs - NBench">
  

  <script src="/nbench/assets/bootstrap/dist/js/jquery-3.2.1.min.js"></script>
<script src="/nbench/assets/bootstrap/dist/js/bootstrap.min.js"></script>

<link href="/nbench/assets/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" type="text/css">
<link href="/nbench/assets/bootstrap/dist/css/stylish-portfolio.css" rel="stylesheet" type = "text/css">
<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
<link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">


  
  <link href="/nbench/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">

  

</head>

  <body   >
    <a href="#content" class="sr-only sr-only-focusable">Skip to main content</a>
    <div id="wrap">
      <header role="banner">
        <nav class="navbar navbar-custom" role="navigation">
	
    <div class="container">
        <div class="navbar-header">
<!--
            <button type="button" class="navbar-toggle" title="toggle navbar" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
-->
            <img src="/nbench/images/logo.jpg" style='max-height:50px;' alt="logo">
            <a class="navbar-brand" href="/nbench/">NBench</a>
        </div>

        <div class="navbar-collapse collapse">
            <ul class="nav navbar-nav">
                <li >
                    <a href="/nbench/news">News</a>
                </li>
                <li >
                    <a href="/nbench/results">Results</a>
                </li>
                <li >
                    <a href="/nbench/datasets">Datasets</a>
                </li>
                <li class="active">
                    <a href="/nbench/programs">Prediction Programs</a>
                </li>
                <li >
                    <a href="/nbench/discussion">Discussion</a>
                </li>
            </ul>
            <ul class="nav navbar-nav navbar-right">
                <li>
                    <a class="subscribe-rss" href="/nbench/atom.xml" title="subscribe via RSS">
                        <span class="visible-xs">RSS</span>
                        <img class="hidden-xs" src="/nbench/images/rss.png" alt="RSS">
                    </a>
                </li>
                
            </ul>
            
                <form class="navbar-form navbar-right" action="https://www.google.com/search" method="GET">
                    <input type="hidden" name="sitesearch" value="chichaumiau.github.io/nbench">
                    <div class="form-group">
                        <input class="form-control" type="text" name="q" placeholder="Search">
                    </div>
                </form>
            
        </div>
    </div>
</nav>


<a id="menu-toggle" href="#" class="btn btn-dark btn-lg toggle"><i class="fa fa-bars"></i></a>
<nav id="sidebar-wrapper">
	<ul class="sidebar-nav">
		<a id="menu-close" href="#" class="btn btn-dark btn-lg pull-right toggle"><i class="fa fa-times"></i></a>
		<li class="sidebar-brand"><a href="https://chichaumiau.github.io/nbench" onclick=$("#menu-close").click();>NBench</a></li>
		<li><a href="#top" onclick=$("#menu-close").click();>Home</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/news" onclick=$("#menu-close").click();>News</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/results" onclick=$("#menu-close").click();>Results</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/datasets" onclick=$("#menu-close").click();>Datasets</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/programs" onclick=$("#menu-close").click();>Programs</a></li>
		<li><a href="https://chichaumiau.github.io/rnapuzzles/discussion" onclick=$("#menu-close").click();>Discussion</a></li>
		<li><a href="https://github.com/RNA-Puzzles" onclick=$("#menu-close").click();>Github</a></li>
	</ul>
</nav>


      </header>
      <div id="main" role="main" class="container">
        <div id="content">
          <div class="row">
  <div class="page-content col-md-9">
    <article role="article">
      
        <header class="page-header">
          
            <p class="meta text-muted text-uppercase">












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2018-01-30T15:04:00+00:00"  data-updated="true" itemprop="datePublished dateCreated">ordinal</time></p>
          
          
            <h1 class="entry-title">
              Programs
              
            </h1>
          
        </header>
      
      <div class="entry-content">
        <div align="center">
<table border="1">
<tr>
<td>Year</td>
<td>Program</td>
<td>Website</td>
</tr>
<tr><td colspan="3"><h3>RBP-sequence based</h3></td></tr>
<tr>
<td>2006</td>
<td>BindN<sup>1</sup></td>
<td><a href="http://bioinfo.ggc.org/bindn/">http://bioinfo.ggc.org/bindn/</a></td>
</tr>
<tr>
<td>2007</td>
<td>RNABindR<sup>2</sup></td>
<td><a href="http://einstein.cs.iastate.edu/RNABindR/">http://einstein.cs.iastate.edu/RNABindR/</a></td>
</tr>
<tr>
<td>2008</td>
<td>PPRInt<sup>3</sup></td>
<td><a href="http://www.imtech.res.in/raghava/pprint/">http://www.imtech.res.in/raghava/pprint/</a></td>
</tr>
<tr>
<td>2008</td>
<td>RNAproB<sup>4</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2008</td>
<td>PRINTR<sup>5</sup></td>
<td><a href="http://210.42.106.80/printr/">http://210.42.106.80/printr/</a></td>
</tr>
<tr>
<td>2008</td>
<td>RISP<sup>6</sup></td>
<td><a href="http://grc.seu.edu.cn/RISP">http://grc.seu.edu.cn/RISP</a></td>
</tr>
<tr>
<td>2009</td>
<td>PiRaNhA<sup>7</sup></td>
<td><a href="http://www.bioinformatics.sussex.ac.uk/PIRANHA">http://www.bioinformatics.sussex.ac.uk/PIRANHA</a></td>
</tr>
<tr>
<td>2010</td>
<td>BindN+<sup>8</sup></td>
<td><a href="http://bioinfo.ggc.org/bindn+/">http://bioinfo.ggc.org/bindn+/</a></td>
</tr>
<tr>
<td>2010</td>
<td>NAPS<sup>9</sup></td>
<td><a href="http://proteomics.bioengr.uic.edu/NAPS/">http://proteomics.bioengr.uic.edu/NAPS/</a></td>
</tr>
<tr>
<td>2010</td>
<td>PRBR<sup>10</sup></td>
<td><a href="http://www.cbi.seu.edu.cn/PRBR/optional.php">http://www.cbi.seu.edu.cn/PRBR/optional.php</a></td>
</tr>
<tr>
<td>2011</td>
<td>Sungwook<sup>11</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2011</td>
<td>SRCPred<sup>12</sup></td>
<td><a href="http://tardis.nibio.go.jp/netasa/srcpred/">http://tardis.nibio.go.jp/netasa/srcpred/</a></td>
</tr>
<tr>
<td>2011</td>
<td>Predict_RBP<sup>13</sup></td>
<td><a href="http://cic.scu.edu.cn/bioinformatics/Predict_RBP.rar">http://cic.scu.edu.cn/bioinformatics/Predict_RBP.rar</a></td>
</tr>
<tr>
<td>2011</td>
<td>meta2<sup>14</sup></td>
<td><a href="https://genesilico.pl/meta2">https://genesilico.pl/meta2</a></td>
</tr>
<tr>
<td>2012</td>
<td>Qian-Zhong<sup>15</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2014</td>
<td>RNABindRPlus<sup>16</sup></td>
<td><a href="http://einstein.cs.iastate.edu/RNABindRPlus/">http://einstein.cs.iastate.edu/RNABindRPlus/</a></td>
</tr>
<tr>
<td>2015</td>
<td>RBRIdent<sup>17</sup></td>
<td><a href="http://166.111.152.91/RBRIdent">http://166.111.152.91/RBRIdent</a></td>
</tr>
<tr><td colspan="3"><h3>RBP-structure based</h3></td></tr>
<tr>
<td>2006</td>
<td>KYG<sup>18</sup></td>
<td><a href="http://cib.cf.ocha.ac.jp/KYG/">http://cib.cf.ocha.ac.jp/KYG/</a></td>
</tr>
<tr>
<td>2008</td>
<td>RsiteDB<sup>19</sup></td>
<td><a href="http://bioinfo3d.cs.tau.ac.il/RsiteDB/">http://bioinfo3d.cs.tau.ac.il/RsiteDB/</a></td>
</tr>
<tr>
<td>2008</td>
<td>DR_bind1<sup>20</sup></td>
<td><a href="http://drbind.limlab.ibms.sinica.edu.tw/">http://drbind.limlab.ibms.sinica.edu.tw/</a></td>
</tr>
<tr>
<td>2009</td>
<td>PRIP<sup>21</sup></td>
<td><a href="http://www.qfab.org/PRIP">http://www.qfab.org/PRIP</a></td>
</tr>
<tr>
<td>2010</td>
<td>OPRA<sup>22</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2011</td>
<td>DRNA<sup>23</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2010</td>
<td>Struct-NB<sup>24</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2010</td>
<td>PRNA<sup>25</sup></td>
<td><a href="http://doc.aporc.org/wiki/PRNA">http://doc.aporc.org/wiki/PRNA</a></td>
</tr>
<tr>
<td>2014</td>
<td>aaRNA<sup>26</sup></td>
<td><a href="http://sysimm.ifrec.osaka-u.ac.jp/aarna/">http://sysimm.ifrec.osaka-u.ac.jp/aarna/</a></td>
</tr>
<tr>
<td>2014</td>
<td>RBRDetector<sup>27</sup></td>
<td><a href="http://ibi.hzau.edu.cn/rbrdetector">http://ibi.hzau.edu.cn/rbrdetector</a></td>
</tr>
<tr>
<td>2014</td>
<td>Xiaoyong<sup>28</sup></td>
<td>NA</td>
</tr>
<tr><td colspan="3"><h3>DBP-sequence based</h3></td></tr>
<tr>
<td>2015</td>
<td>RBscore<sup>29</sup></td>
<td><a href="http://ahsoka.u-strasbg.fr/rbscore/">http://ahsoka.u-strasbg.fr/rbscore/</a></td>
</tr>
<tr>
<td>2004</td>
<td>DBS-Pred<sup>30</sup></td>
<td><a href="http://www.abren.net/dbs-pred/">http://www.abren.net/dbs-pred/</a></td>
</tr>
<tr>
<td>2005</td>
<td>DBS-PSSM<sup>31</sup></td>
<td><a href="http://www.abren.net/dbs-pssm/">http://www.abren.net/dbs-pssm/</a></td>
</tr>
<tr>
<td>2006</td>
<td>DNABindR<sup>32</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2007</td>
<td>DISIS<sup>33</sup></td>
<td><a href="http://cubic.bioc.columbia.edu/services/disis">http://cubic.bioc.columbia.edu/services/disis</a></td>
</tr>
<tr>
<td>2007</td>
<td>DP-Bind<sup>34</sup></td>
<td><a href="http://lcg.rit.albany.edu/dp-bind/">http://lcg.rit.albany.edu/dp-bind/</a></td>
</tr>
<tr>
<td>2009</td>
<td>ProteDNA<sup>35</sup></td>
<td><a href="http://protedna.csbb.ntu.edu.tw/">http://protedna.csbb.ntu.edu.tw/</a></td>
</tr>
<tr>
<td>2009</td>
<td>DbindR<sup>36</sup></td>
<td><a href="http://www.cbi.seu.edu.cn/DBindR/DBindR.htm">http://www.cbi.seu.edu.cn/DBindR/DBindR.htm</a></td>
</tr>
<tr>
<td>2009</td>
<td>SDCPred<sup>37</sup></td>
<td><a href="http://sdcpred.netasa.org/">http://sdcpred.netasa.org/</a></td>
</tr>
<tr>
<td>2014</td>
<td>Byungkyu<sup>38</sup></td>
<td>NA</td>
</tr>
<tr><td colspan="3"><h3>DBP-structure based</h3></td></tr>
<tr>
<td>1999</td>
<td>Hidetoshi<sup>39</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2003</td>
<td>Susan<sup>40</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2005</td>
<td>DBS-kernel<sup>41</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2005</td>
<td>Pro-dna<sup>42</sup></td>
<td><a href="http://bioinformatics.bioengr.uic.edu/pro-dna/">http://bioinformatics.bioengr.uic.edu/pro-dna/</a></td>
</tr>
<tr>
<td>2005</td>
<td>PreDs<sup>43</sup></td>
<td><a href="http://pre-s.protein.osaka-u.ac.jp/&#126;preds/">http://pre-s.protein.osaka-u.ac.jp/&#126;preds/</a></td>
</tr>
<tr>
<td>2007</td>
<td>DISPLAR<sup>44</sup></td>
<td><a href="http://pipe.scs.fsu.edu/displar.html">http://pipe.scs.fsu.edu/displar.html</a></td>
</tr>
<tr>
<td>2007</td>
<td>DR_bind1<sup>45</sup></td>
<td><a href="http://drbind.limlab.ibms.sinica.edu.tw/">http://drbind.limlab.ibms.sinica.edu.tw/</a></td>
</tr>
<tr>
<td>2008</td>
<td>DBD-Hunter<sup>46</sup></td>
<td><a href="http://cssb.biology.gatech.edu/skolnick/webservice/DBD-Hunter/">http://cssb.biology.gatech.edu/skolnick/webservice/DBD-Hunter/</a></td>
</tr>
<tr>
<td>2010</td>
<td>DNABINDPROT<sup>47</sup></td>
<td><a href="http://www.prc.boun.edu.tr/appserv/prc/dnabindprot/">http://www.prc.boun.edu.tr/appserv/prc/dnabindprot/</a></td>
</tr>
<tr>
<td>2011</td>
<td>metaDBSite<sup>48</sup></td>
<td><a href="http://projects.biotec.tu-dresden.de/metadbsite/">http://projects.biotec.tu-dresden.de/metadbsite/</a></td>
</tr>
<tr>
<td>2011</td>
<td>Xiong<sup>49</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2012</td>
<td>Sucharita<sup>50</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2013</td>
<td>Duo-Duo<sup>51</sup></td>
<td>NA</td>
</tr>
<tr>
<td>2013</td>
<td>PreDNA<sup>52</sup></td>
<td><a href="http://202.207.14.178/predna/">http://202.207.14.178/predna/</a></td>
</tr>
<tr>
<td>2013</td>
<td>DNABind<sup>53</sup></td>
<td><a href="http://mleg.cse.sc.edu/DNABind/">http://mleg.cse.sc.edu/DNABind/</a></td>
</tr>
<tr>
<td>2014</td>
<td>Bi-Qing<sup>54</sup></td>
<td>NA</td>
</tr>
</table>
</div>


<h2>References:</h2>

<ol>
<li>Wang, L.J. and Brown, S.J. (2006) BindN: a web-based tool for efficient prediction of DNA and RNA binding sites in amino acid sequences. Nucleic Acids Research, 34, W243-W248.</li>
<li>Terribilini, M., Sander, J.D., Lee, J.H., Zaback, P., Jernigan, R.L., Honavar, V. and Dobbs, D. (2007) RNABindR: a server for analyzing and predicting RNA-binding sites in proteins. Nucleic Acids Research, 35, W578-W584.</li>
<li>Kumar, M., Gromiha, A.M. and Raghava, G.P.S. (2008) Prediction of RNA binding sites in a protein using SVM and PSSM profile. Proteins, 71, 189-194.</li>
<li>Cheng, C.W., Su, E.C.Y., Hwang, J.K., Sung, T.Y. and Hsu, W.L. (2008) Predicting RNA-binding sites of proteins using support vector machines and evolutionary information. Bmc Bioinformatics, 9.</li>
<li>Wang, Y., Xue, Z., Shen, G. and Xu, J. (2008) PRINTR: Prediction of RNA binding sites in proteins using SVM and profiles. Amino Acids, 35, 295-302.</li>
<li>Tong, J., Jiang, P. and Lu, Z.H. (2008) RISP: A web-based server for prediction of RNA-binding sites in proteins. Comput Meth Prog Bio, 90, 148-153.</li>
<li>Murakami, Y., Spriggs, R.V., Nakamura, H. and Jones, S. (2010) PiRaNhA: a server for the computational prediction of RNA-binding residues in protein sequences. Nucleic Acids Research, 38, W412-W416.</li>
<li>Wang, L.J., Huang, C.Y., Yang, M.Q. and Yang, J.Y. (2010) BindN plus for accurate prediction of DNA and RNA-binding residues from protein sequence features. Bmc Syst Biol, 4.</li>
<li>Carson, M.B., Langlois, R. and Lu, H. (2010) NAPS: a residue-level nucleic acid-binding prediction server. Nucleic Acids Research, 38, W431-W435.</li>
<li>Ma, X., Guo, J., Wu, J.S., Liu, H.D., Yu, J.F., Xie, J.M. and Sun, X.A. (2011) Prediction of RNA-binding residues in proteins from primary sequence using an enriched random forest model with a novel hybrid feature. Proteins, 79, 1230-1239.</li>
<li>Choi, S. and Han, K. (2011) Prediction of RNA-binding amino acids from protein and RNA sequences. Bmc Bioinformatics, 12.</li>
<li>Fernandez, M., Kumagai, Y., Standley, D.M., Sarai, A., Mizuguchi, K. and Ahmad, S. (2011) Prediction of dinucleotide-specific RNA-binding sites in proteins. Bmc Bioinformatics, 12.</li>
<li>Wang, C.C., Fang, Y.P., Xiao, J.M. and Li, M.L. (2011) Identification of RNA-binding sites in proteins by integrating various sequence information. Amino Acids, 40, 239-248.</li>
<li>Puton, T., Kozlowski, L., Tuszynska, I., Rother, K. and Bujnicki, J.M. (2012) Computational methods for prediction of protein-RNA interactions. J Struct Biol, 179, 261-268.</li>
<li>Li, T. and Li, Q.Z. (2012) Annotating the protein-RNA interaction sites in proteins using evolutionary information and protein backbone structure. J Theor Biol, 312, 55-64.</li>
<li>Walia, R.R., Xue, L.C., Wilkins, K., El-Manzalawy, Y., Dobbs, D. and Honavar, V. (2014) RNABindRPlus: A Predictor that Combines Machine Learning and Sequence Homology-Based Methods to Improve the Reliability of Predicted RNA-Binding Residues in Proteins. Plos One, 9.</li>
<li>Xiong, D., Zeng, J. and Gong, H. (2015) RBRIdent: An algorithm for improved identification of RNA-binding residues in proteins from primary sequences. Proteins, 83, 1068-1077.</li>
<li>Kim, O.T.P., Yura, K. and Go, N. (2006) Amino acid residue doublet propensity in the protein-RNA interface and its application to RNA interface prediction. Nucleic Acids Research, 34, 6450-6460.</li>
<li>Shulman-Peleg, A., Shatsky, M., Nussinov, R. and Wolfson, H.J. (2008) Prediction of interacting single-stranded RNA bases by protein-binding patterns. J Mol Biol, 379, 299-316.</li>
<li>Chen, Y.C. and Lim, C. (2008) Predicting RNA-binding sites from the protein structure based on electrostatics, evolution and geometry. Nucleic Acids Res, 36.</li>
<li>Maetschke, S.R. and Yuan, Z. (2009) Exploiting structural and topological information to improve prediction of RNA-protein binding sites. Bmc Bioinformatics, 10.</li>
<li>Perez-Cano, L. and Fernandez-Recio, J. (2010) Optimal Protein-RNA Area, OPRA: A propensity-based method to identify RNA-binding sites on proteins. Proteins, 78, 25-35.</li>
<li>Zhao, H.Y., Yang, Y.D. and Zhou, Y.Q. (2011) Structure-based prediction of RNA-binding domains and RNA-binding sites and application to structural genomics targets. Nucleic Acids Research, 39, 3017-3025.</li>
<li>Towfic, F., Caragea, C., Gemperline, D.C., Dobbs, D. and Honavar, V. (2010) Struct-NB: predicting protein-RNA binding sites using structural features. Int J Data Min Bioin, 4, 21-43.</li>
<li>Liu, Z.P., Wu, L.Y., Wang, Y., Zhang, X.S. and Chen, L.N. (2010) Prediction of protein-RNA binding sites by a random forest method with combined features. Bioinformatics, 26, 1616-1622.</li>
<li>Li, S., Yamashita, K., Amada, K.M. and Standley, D.M. (2014) Quantifying sequence and structural features of protein-RNA interactions. Nucleic Acids Res.</li>
<li>Yang, X.X., Deng, Z.L. and Liu, R. (2014) RBRDetector: improved prediction of binding residues on RNA-binding protein structures using complementary feature- and template-based strategies. Proteins, 82, 2455-2471.</li>
<li>Pan, X., Zhu, L., Fan, Y.-X. and Yan, J. (2014) Predicting protein-RNA interaction amino acids using random forest based on submodularity subset selection. Computational Biology and Chemistry, 53, Part B, 324-330.</li>
<li>Miao, Z. and Westhof, E. (2015) Prediction of nucleic acid binding probability in proteins: a neighboring residue network based score. Nucleic Acids Res.</li>
<li>Ahmad, S., Gromiha, M.M. and Sarai, A. (2004) Analysis and prediction of DNA-binding proteins and their binding residues based on composition, sequence and structural information. Bioinformatics, 20, 477-486.</li>
<li>Ahmad, S. and Sarai, A. (2005) PSSM-based prediction of DNA binding sites in proteins. Bmc Bioinformatics, 6.</li>
<li>Yan, C.H., Terribilini, M., Wu, F.H., Jernigan, R.L., Dobbs, D. and Honavar, V. (2006) Predicting DNA-binding sites of proteins from amino acid sequence. Bmc Bioinformatics, 7.</li>
<li>Ofran, Y., Mysore, V. and Rost, B. (2007) Prediction of DNA-binding residues from sequence. Bioinformatics, 23, I347-I353.</li>
<li>Hwang, S., Gou, Z.K. and Kuznetsov, I.B. (2007) DP-Bind: a Web server for sequence-based prediction of DNA-binding residues in DNA-binding proteins. Bioinformatics, 23, 634-636.</li>
<li>Chu, W.Y., Huang, Y.F., Huang, C.C., Cheng, Y.S., Huang, C.K. and Oyang, Y.J. (2009) ProteDNA: a sequence-based predictor of sequence-specific DNA-binding residues in transcription factors. Nucleic Acids Res, 37, W396-W401.</li>
<li>Wu, J.S., Liu, H.D., Duan, X.Y., Ding, Y., Wu, H.T., Bai, Y.F. and Sun, X. (2009) Prediction of DNA-binding residues in proteins from amino acid sequences using a random forest model with a hybrid feature. Bioinformatics, 25, 30-35.</li>
<li>Andrabi, M., Mizuguchi, K., Sarai, A. and Ahmad, S. (2009) Prediction of mono- and di-nucleotide-specific DNA-binding sites in proteins using neural networks. Bmc Struct Biol, 9.</li>
<li>Park, B., Im, J., Tuvshinjargal, N., Lee, W. and Han, K. (2014) Sequence-based prediction of protein-binding sites in DNA: Comparative study of two SVM models. Comput Meth Prog Bio, 117, 158-167.</li>
<li>Kono, H. and Sarai, A. (1999) Structure-based prediction of DNA target sites by regulatory proteins. Proteins-Structure Function And Genetics, 35, 114-131.</li>
<li>Jones, S., Shanahan, H.P., Berman, H.M. and Thornton, J.M. (2003) Using electrostatic potentials to predict DNA-binding sites on DNA-binding proteins. Nucleic Acids Res, 31, 7189-7198.</li>
<li>Bhardwaj, N., Langlois, R.E., Zhao, G.J. and Lu, H. (2005) Kernel-based machine learning protocol for predicting DNA-binding proteins. Nucleic Acids Research, 33, 6486-6493.</li>
<li>Bhardwaj, N. and Lu, H. (2007) Residue-level prediction of DNA-binding sites and its application on DNA-binding protein predictions. Febs Lett, 581, 1058-1066.</li>
<li>Tsuchiya, Y., Kinoshita, K. and Nakamura, H. (2005) PreDs: a server for predicting dsDNA-binding site on protein molecular surfaces. Bioinformatics, 21, 1721-1723.</li>
<li>Tjong, H. and Zhou, H.X. (2007) DISPLAR: an accurate method for predicting DNA-binding sites on protein surfaces. Nucleic Acids Research, 35, 1465-1477.</li>
<li>Chen, Y.C., Wu, C.Y. and Lim, C. (2007) Predicting DNA-binding amino acid residues from electrostatic stabilization upon mutation to Asp/Glu and evolutionary conservation. Proteins, 67, 671-680.</li>
<li>Gao, M. and Skolnick, J. (2008) DBD-Hunter: a knowledge-based method for the prediction of DNA-protein interactions. Nucleic Acids Research, 36, 3978-3992.</li>
<li>Ozbek, P., Soner, S., Erman, B. and Haliloglu, T. (2010) DNABINDPROT: fluctuation-based predictor of DNA-binding residues within a network of interacting residues. Nucleic Acids Res, 38, W417-W423.</li>
<li>Si, J.N., Zhang, Z.M., Lin, B.Y., Schroeder, M. and Huang, B.D. (2011) MetaDBSite: a meta approach to improve protein DNA-binding sites prediction. Bmc Syst Biol, 5.</li>
<li>Xiong, Y., Liu, J. and Wei, D.Q. (2011) An accurate feature-based method for identifying DNA-binding residues on protein surfaces. Proteins, 79, 509-517.</li>
<li>Dey, S., Pal, A., Guharoy, M., Sonavane, S. and Chakrabarti, P. (2012) Characterization and prediction of the binding site in DNA-binding proteins: improvement of accuracy by combining residue composition, evolutionary conservation and structural parameters. Nucleic Acids Research, 40, 7150-7161.</li>
<li>Wang, D.D., Li, T.H., Sun, J.M., Li, D.P., Xiong, W.W., Wang, W.Y. and Tang, S.N. (2013) Shape string: A new feature for prediction of DNA-binding residues. Biochimie, 95, 354-358.</li>
<li>Li, T., Li, Q.Z., Liu, S., Fan, G.L., Zuo, Y.C. and Peng, Y. (2013) PreDNA: accurate prediction of DNA-binding sites in proteins by integrating sequence and geometric structure information. Bioinformatics, 29, 678-685.</li>
<li>Liu, R. and Hu, J. (2013) DNABind: a hybrid algorithm for structure-based prediction of DNA-binding residues by combining machine learning- and template-based approaches. Proteins, 81, 1885-1899.</li>
<li>Li, B.Q., Feng, K.Y., Ding, J. and Cai, Y.D. (2014) Predicting DNA-binding sites of proteins based on sequential and 3D structural information. Mol Genet Genomics, 289, 489-499.</li>
</ol>


      </div>
      
        <footer>
          <p class="meta text-muted">
            
            












<span class="glyphicon glyphicon-calendar"></span> <time datetime="2018-01-30T15:04:00+00:00"  data-updated="true" itemprop="datePublished dateCreated">ordinal</time>
            
          </p>
          
            <div class="sharing">
  
  <a href="//twitter.com/share" class="twitter-share-button" data-url="http://chichaumiau.github.io/nbench/programs/index.html" data-via="chichaumiau" data-counturl="http://chichaumiau.github.io/nbench/programs/index.html" >Tweet</a>
  
  
  
</div>

          
        </footer>
      
    </article>
    
      <section>
        <h2>Comments</h2>
        <div id="disqus_thread" aria-live="polite"><noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
</div>
      </section>
    
  </div>

  
    <aside class="sidebar col-md-3">
      
        <section>
  <h1>NBench</h1>
  <p>Benchmark of nucleic acid binding site prediction methods.</p>
	<a href="https://github.com/chichaumiau"><i class="fa fa-github-square fa-fw fa-2x"></i></a>
	<a href="mailto:chichaumiau@gmail.com"><i class="fa fa-envelope-square fa-fw fa-2x"></i></a>
	<a href="https://twitter.com/chichaumiau"><i class="fa fa-twitter-square fa-fw fa-2x"></i></a>     
	<a href="https://plus.google.com/u/0/104588354121735738405"><i class="fa fa-google-plus-square fa-fw fa-2x"></i></a>
	<a href=""><i class="fa fa-coffee fa-fw fa-2x"></i></a>
</section>
<section class="panel panel-info">
  <div class="panel-heading">
    <h3 class="panel-title">Recent Posts</h3>
  </div>
  
  <div id="recent_posts" class="list-group">
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/moonlighting-proteins/">Moonlighting Proteins</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/nucleic-acid-binding-energy-funnel-on-protein-surface/">Nucleic Acid Binding Energy Funnel on Protein Surface</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/risky-non-orthogonal-features/">Risky Non-orthogonal Features</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/theoretical-defect-of-slide-window/">Theoretical Defect of Slide-window</a>
    
    <a class="list-group-item " href="/nbench/blog/2018/01/30/discriminate-rna-binding-residues-from-dna-binding-residues/">Discriminate RNA Binding Residues From DNA Binding Residues</a>
    
  </div>
</section>

<section class="panel panel-info clearfix">
  <div class="panel-heading">
      <h3 class="panel-title">GitHub Repos</h3>
  </div>
  <div class="list-group" id="gh_repos">
    <p class="loading">Status updating...</p>
  </div>
  
    <div class="gh-profile-link pull-right text-muted">
      <a href="https://github.com/chichaumiau">@chichaumiau</a> on GitHub
    </div>
  
  <script type="text/javascript">
    $(document).ready(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'chichaumiau',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/nbench/javascripts/github.js" type="text/javascript"> </script>
</section>

<section class="panel panel-info clearfix">
  <div class="panel-heading">
      <h3 class="panel-title">Categories</h3>
  </div>
    <div class='list-group-item'><a href='/nbench/blog/categories/discussion/'>discussion (6)</a></div>
<div class='list-group-item'><a href='/nbench/blog/categories/table/'>table (1)</a></div>


</section>

<section class="googleplus panel panel-default">
  <div class="panel-body">
    <h2>
      <a href="https://plus.google.com/104588354121735738405?rel=author">
        <img src="http://www.google.com/images/icons/ui/gprofile_button-32.png" width="32" height="32">
        Google+
      </a>
    </h2>
  </div>
</section>





      
    </aside>
  
</div>

        </div>
      </div>
    </div>
    <footer role="contentinfo"><div class="container">
    <p class="text-muted credits" align="center">
  Copyright &copy; 2018 - <a href="mailto:chichaumiau@gmail.com">Chichau Miau</a>
  <a id="to-top" href="#top" class="btn btn-dark btn-lg"><i class="fa fa-chevron-up fa-fw fa-1x"></i></a>
</p>

</div>
</footer>
    

<script type="text/javascript">
      var disqus_shortname = 'rna-puzzles';
      
        
        // var disqus_developer = 1;
        var disqus_identifier = 'http://chichaumiau.github.io/nbench/programs/index.html';
        var disqus_url = 'http://chichaumiau.github.io/nbench/programs/index.html';
        var disqus_script = 'embed.js';
      
    (function () {
      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
      dsq.src = '//' + disqus_shortname + '.disqus.com/' + disqus_script;
      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    }());
</script>







  <script type="text/javascript">
    (function(){
      var twitterWidgets = document.createElement('script');
      twitterWidgets.type = 'text/javascript';
      twitterWidgets.async = true;
      twitterWidgets.src = '//platform.twitter.com/widgets.js';
      document.getElementsByTagName('head')[0].appendChild(twitterWidgets);
    })();
  </script>


<script src="/nbench/assets/bootstrap/dist/js/bootstrap.min.js"></script>
<script src="/nbench/javascripts/modernizr.js"></script>


<!-- Custom Theme JavaScript -->
<script>
// Closes the sidebar menu
$("#menu-close").click(function(e) {
	e.preventDefault();
	$("#sidebar-wrapper").toggleClass("active");
});
// Opens the sidebar menu
$("#menu-toggle").click(function(e) {
	e.preventDefault();
	$("#sidebar-wrapper").toggleClass("active");
});
// Scrolls to the selected menu item on the page
$(function() {
	$('a[href*=#]:not([href=#],[data-toggle],[data-target],[data-slide])').click(function() {
		if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') || location.hostname == this.hostname) {
			var target = $(this.hash);
			target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
			if (target.length) {
				$('html,body').animate({
					scrollTop: target.offset().top
				}, 1000);
				return false;
			}
		}
	});
});
//#to-top button appears after scrolling
var fixed = false;
$(document).scroll(function() {
	if ($(this).scrollTop() > 250) {
		if (!fixed) {
			fixed = true;
			// $('#to-top').css({position:'fixed', display:'block'});
			$('#to-top').show("slow", function() {
				$('#to-top').css({
					position: 'fixed',
					display: 'block'
				});
			});
		}
	} else {
		if (fixed) {
			fixed = false;
			$('#to-top').hide("slow", function() {
				$('#to-top').css({
					display: 'none'
				});
			});
		}
	}
});
</script>


  </body>
</html>
